scrunch/datasets.py

```diff
@@ -3460,7 +3485,10 @@ def backfill(self):
             "variables": variables_expr,
             "filter": self.rows_expr
         }
-        self.dataset.resource.table.post(update_expr)
+        with NoExclusion(self.dataset) as ds:
+            print("Current excl")
+            print(ds.resource.exclusion)
```

Comment:
Should we remove those couple of prints?

Reply:
Done